elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -75,10 +75,13 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
   import RestlasticSearchClient.ReturnTypes._
 
   def ready: Boolean = endpointProvider.ready
-  def query(index: Index, tpe: Type, query: QueryRoot): Future[SearchResponse] = {
+  def query(index: Index, tpe: Type, query: QueryRoot, rawJsonStr: Option[Boolean] = Some(true)): Future[SearchResponse] = {
```

Comment:
I don't really have a preference for one verse the other. One is shorter & cleaner, the other clearly tells me it is an option. Updated to `ture` as suggested.